### DISCUSSION TITLE
Can open the current file in the tree directory?

### DISCUSSION THREAD

**Discussion by mrluisfer** [nvim-tree/nvim-tree.lua#1900] [Q&A] [answered]
I recently started use the `nvim-tree.lua` plugin to have a good directory tree, but reading the "documentation" I couldn't find (or I didn't see any) a property to open it so that the file appears in the current directory

![Captura de pantalla 2023-01-08 a la(s) 8 32 01](https://user-images.githubusercontent.com/65029792/211201973-5a116f3b-eab8-4075-86a8-9e7517b3194a.png)

**Comment by gegoune** [comment, marked as answer]
I am not entirely sure what you mean exactly, but perhaps that is what you are after: `:h nvim-tree.update_focused_file`?

**Reply by mrluisfer** [reply]
Yep, that was just the option I needed
Thanks!
